scene/gui/texture_progress_bar.cpp

```diff
@@ -81,15 +81,19 @@ bool TextureProgressBar::get_nine_patch_stretch() const {
 Size2 TextureProgressBar::get_minimum_size() const {
 	if (nine_patch_stretch) {
 		return Size2(stretch_margin[SIDE_LEFT] + stretch_margin[SIDE_RIGHT], stretch_margin[SIDE_TOP] + stretch_margin[SIDE_BOTTOM]);
-	} else if (under.is_valid()) {
-		return under->get_size();
-	} else if (over.is_valid()) {
-		return over->get_size();
-	} else if (progress.is_valid()) {
-		return progress->get_size();
+	} else {
+		Size2 size = Size2(1, 1);
+		if (under.is_valid()) {
+			size = under->get_size();
+		}
+		if (progress.is_valid() && progress->get_size() > size) {
+			size = progress->get_size();
+		}
+		if (over.is_valid() && over->get_size() > size) {
+			size = over->get_size();
```

Comment:
```suggestion
		if (under.is_valid()) {
			size = size.max(under->get_size());
		}
		if (progress.is_valid()) {
			size = size.max(progress->get_size());
		}
		if (over.is_valid()) {
			size = size.max(over->get_size());
```
This will replace a size of `(5, 5)` with a size of `(6, 1)`, I'm not sure that's desired, I'd say to use the element wise max instead